.gitignore

```diff
@@ -1,3 +1,52 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
please remove these local changes. developers are expected to have these exclusions setup locally